modules/users/client/views/admin/view-applicants.html

```diff
@@ -0,0 +1,38 @@
+<section>
+  <div class="page-header">
+    <div class="row">
+      <div class="col-md-4">
+        <h1>Users</h1>
+      </div>
+      <div class="col-md-4" style="margin-top: 2em">
+        <input class="form-control col-md-4" type="text" ng-model="vm.search" placeholder="Search" ng-change="vm.figureOutItemsToDisplay()" />
+      </div>
+    </div>
+  </div>
+  <div class="list-group">
+    <table class="table table-hover applicantsTable">
+    	<thead>
+    		<tr class="first-row">
+    			<td width="25%">Name</td>
```

Comment:
width should be handled automatically, but if we must specify that it takes 25%, then we should put this in css